### ISSUE TITLE
Project-level modules cannot extend other project-level modules

### ISSUE THREAD

**Issue by Danny-weebo** [punkave/apostrophe#182]
Hey, I'm trying to create a custom module that will sub class `apostrophe-snippet` and to use my new module as the extended one when creating a new type module (in app.js).

Apostrophe-snippet > comments -> some Other Module

this is my code (based on apostrophe-blog / apostrophe-events) 
http://pastebin.com/CkLbjbm8

now the problem is, every time I create new module that extends from `comments` I get this error 
```
WARNING: unable to ensure text index, apostrophe:migrate can fix that
Unable to find C:\web/lib/modules/articles_discourse/index.js, or an error took place in that file (see below). Perhaps you forgot to npm install something, or you forgot to set the extend property for this module.

C:\web\node_modules\apostrophe-site\index.js:392
            throw e;
                  ^
Error: Cannot find module 'C:\web/lib/modules/articles_discourse/index.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at C:\web\node_modules\apostrophe-site\index.js:389:23
    at iterate (C:\web\node_modules\apostrophe-site\node_modules\async\lib\async.js:149:13)
    at C:\web\node_modules\apostrophe-site\node_modules\async\lib\async.js:160:25
    at C:\web\node_modules\apostrophe-site\index.js:486:16
    at C:\web\node_modules\apostrophe-snippets\index.js:2295:42
    at process._tickCallback (node.js:415:13)
```

why is that ? how come apostrophe-blog doesn't require `index.js` in the module ? can't find what I'm missing.. thanks

@israelidanny

**Comment by israelidanny** [comment]
Any chance you could take a look at it already? :) Would be very greatly appreciated. We're sorta clueless and there's a whole project that can't ship only because of this little setback.

**Comment by boutell** [comment]
Want to send me a link to a tarball? tom@punkave.com

(Should be a tarball of a project where you've done everything right as far
as you know, with very clear instructions to reproduce the problem.)

On Sat, Jan 3, 2015 at 4:56 PM, Danny Povolotski <notifications@github.com>
wrote:

> Any chance you could take a look at it already? :) Would be very greatly
> appreciated. We're sorta clueless and there's a whole project that can't
> ship only because of this little setback.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/punkave/apostrophe/issues/182#issuecomment-68611189>.
>



-- 


*THOMAS BOUTELL, *DEV & OPS
P'UNK AVENUE | (215) 755-1330  |  punkave.com

**Comment by israelidanny** [comment]
It's 00:35, so without being able to reach the other @Danny-weebo , I sent you what I think is his latest attempt.

**Comment by benirose** [comment]
@boutell were you able to help the Danny's out with this one?

On another note, doesn't moog-require handle double project-level subclassing?

**Comment by boutell** [comment]
The original title of this issue will not be addressed in 0.5. There are
reasonable workarounds. It is allowed in 0.6.

The rest of this issue was unrelated.
